sentinel-2/ozone_column_over_snow/script.js

```diff
@@ -0,0 +1,101 @@
+function setup () {
+  return {
+    input: [
+      'sunZenithAngles',
+      'viewZenithMean',
+      'B01',
+      'B03',
+      'B8A',
+      'dataMask'
+    ],
+    output: { bands: 4, sampleType: 'AUTO' }
+  }
+}
+
+function degrees_to_radians (degrees) {
+  // Convert degrees to radians
+  return degrees * (Math.PI / 180)
+}
+
+function amf (sza, vza) {
+  // Compute the air mass factor (AMF)
+  // sza in radians
+  // vza in radians
+
+  let epsilon = Math.cos(sza)
+  let eta = Math.cos(vza)
+
+  // Calculate AMF
+  var m = 1 / epsilon + 1 / eta
+
+  return m
+}
+
+function calcK (reflA, reflB, L) {
+  // Compute K from Eq. 5
+
+  // ab and cAbsB are constants from Table 2
+  const ab = 7.48 * Math.pow(10, -4)
```

Comment:
It is nicer if you write such numbers in a scientific notation:
```javascript
const ab = 7.48e-4
```
You can even put all constants outside of functions so that they will be defined only once and not for every pixel again.

Reply:
Done